src/Http/Controllers/OtterViewController.php

```diff
@@ -17,10 +17,12 @@ public function __construct(Request $request)
             $this->baseResourceName = Otter::getClassNameFromRouteName($this->resourceName);
             $this->resource = $this->resourceNamespace.$this->baseResourceName;
             $this->prettyResourceName = str_singular(ucwords(str_replace('_', ' ', $this->resourceName)));
+	        
+            if (class_exists($this->resource))
+                $this->resourceRouteKeyName = $this->resource::$routeKeyName;
             /* @var TYPE_NAME $model */
-            $this->modelName = ($request->is('otter')) ? null : $this->resource::$model;
+            $this->modelName = ($request->is(config('otter.uri', 'otter'))) ? null : $this->resource::$model;
```

Comment:
```suggestion
            $this->modelName = ($request->is(config('otter.path', 'otter'))) ? null : $this->resource::$model;
```

Let's use they word 'path' instead, following telescope's config values